rpcs3/rpcs3qt/log_frame.cpp

```diff
@@ -493,24 +501,33 @@ void log_frame::UpdateUI()
 					m_tty->appendPlainText(tty_text);
 				}
 
+				if (buf[end_line] == '\n')
+				{
+					// Record line history to skip it later only if it did actually end
+					m_tty_lines++;
+				}
+
 				// if we mark text from right to left we need to swap sides (start is always smaller than end)
 				if (sel_pos < sel_end)
 				{
 					std::swap(sel_start, sel_end);
 				}
 
 				// reset old text cursor and selection
-				text_cursor.setPosition(sel_start);
-				text_cursor.setPosition(sel_end, QTextCursor::KeepAnchor);
-				m_tty->setTextCursor(text_cursor);
+				if (sel_end < buf.size())
```

Comment:
this doesn't make sense. the sel_end cannot have gotten out of selectable range.
I think this is a remnant of your "clear" code and should be removed

Reply:
Fixed.